cilium/lb/main.go

```diff
@@ -344,19 +380,34 @@ func cliUpdateService(ctx *cli.Context) {
 	}
 
 	for _, backend := range backendList {
-		beAddr, err := net.ResolveTCPAddr("tcp", backend)
+		tmp := strings.Split(backend, ",")
```

Comment:
Extended the comment for backend. "Usage: "Backend address and port, optional weight"